docs/reference/troubleshooting/troubleshooting-searches.asciidoc

```diff
@@ -0,0 +1,291 @@
+
+[[troubleshooting-searches]]
+== Troubleshooting searches
+
+When you query your data, Elasticsearch may return an error, no search results,
+or results in an unexpected order. This guide describes how to troubleshoot
+searches.
+
+[discrete]
+[[troubleshooting-searches-exists]]
+=== Ensure the data stream, index, or alias exists
+
+Elasticsearch returns an `index_not_found_exception` when the data stream, index
+or alias you try to query does not exist. This can happen when you misspell the
+name or when the data has been indexed to a different data stream or index.
```

Comment:
would it make sense to mention the indices options `ignore_unavailable` etc. or does it complicate things?

Reply:
I've added a note about retrieving partial search results by setting `ignore_unavailable` to`true`.